### ISSUE TITLE
Travis CI questions

### ISSUE THREAD

**Issue by mauzey1** [PCMDI/cmor#595]
The CMOR repo is using both CircleCI and Travis CI for continuous integration.  CircleCI and Travis CI both generate nightly Linux builds for CMOR, while OSX builds are only done in CircleCI.  Was there a reason for having both CI services make Linux builds aside from redundancy?  The Linux builds seem to be identical.  Should we also have OSX builds being made by Travis?

@doutriaux1 @dnadeau4 Could you share your reasoning behind having both CircleCI and Travis for continuous integration?  Thank you.

**Comment by taylor13** [comment]
@doutriaux1 @dnadeau4 Yes, please tell us whether both are still needed.  Your sage advice would be much appreciated.

**Comment by mauzey1** [comment]
Another issue I would like to bring up is the running of the OSX builds on CircleCI taking a long time.  The Linux builds all get executed in parallel but the OSX builds run one at a time.  This leads to builds on CircleCI taking almost an hour to complete the Python 2.7, 3.6, 3.7, and 3.8 OSX builds.

Is there a way to get parallel builds on CircleCI for the OSX builds?  Maybe Travis could provide the OSX builds.

**Comment by durack1** [comment]
@muryanto1 with all the QA work you’ve been doing with CDAT, so you have a perspective to add here?

**Comment by muryanto1** [comment]
For CDAT, we are building both osx and linux flavors in circleci. We do not build on Travis anymore (as far as I know). We subscribe to performance plan with circleci so that we can run jobs in parallel. It reduces the build time tremendously since all jobs can run in parallel. The performance plan is for all projects under CDAT organizations (all projects under https://www.github.com/cdat/).

**Comment by durack1** [comment]
Thanks @muryanto1, so you know what the cost is for the performance plan?

**Comment by muryanto1** [comment]
I recall it is $10K/year. You can talk to circleci folks and they will show you your usage pattern and recommend a plan for you.

**Comment by durack1** [comment]
I wonder if they would allow us to tag on the PCMDI organization to the CDAT organization, a couple of repos.

From: Lina Muryanto <notifications@github.com>

I recall it is $10K/year. You can talk to circleci folks and they will show you your usage pattern and recommend a plan for you.

**Comment by muryanto1** [comment]
@durack1 I will find out if we can tag pcmdi onto the cdat plan.

**Comment by durack1** [comment]
@muryanto1 perfect, it's a long shot, but would be great if it works out

**Comment by mauzey1** [comment]
I plan to remove the Travis CI script from this repo since I don't see a need for it anymore.  CircleCI is already handling the Linux builds and has a better interface.  I will remove .travis.yml in the CMOR 3.6.1 branch.

**Comment by durack1** [comment]
@mauzey1 thanks, a nice little cleanup
